src/components/Common/MultipleComboBox.tsx

```diff
@@ -0,0 +1,279 @@
+import {
+  autoUpdate,
+  flip,
+  offset,
+  size,
+  useFloating,
+} from '@floating-ui/react';
+import { IconX } from '@tabler/icons-react';
+import {
+  FC,
+  RefObject,
+  createElement,
+  useLayoutEffect,
+  useMemo,
+  useState,
+} from 'react';
+
+import { useTranslation } from 'next-i18next';
+
+import classNames from 'classnames';
+
+import { Translation } from '@/src/types/translation';
+
+import { useCombobox, useMultipleSelection } from 'downshift';
+
+interface Props<T> {
+  items: T[];
+  initialSelectedItems?: T[];
+  label?: string;
+  placeholder?: string;
+  notFoundPlaceholder?: string;
+  itemRow?: FC<{ item: T }>;
+  selectedItemRow?: FC<{ item: T }>;
+  disabled?: boolean;
+  isHideDropdown?: boolean;
+  getItemLabel: (item: T) => string;
+  getItemValue: (item: T) => string;
+  getFilteredItems?: (
+    items: T[],
+    inputValue: string | undefined,
+    getItemLabel: (item: T) => string,
+    selectedItems?: T[],
+  ) => T[];
```

Comment:
I don't think we need method as property (we have everything to execute it inside). 
![image](https://github.com/epam/ai-dial-chat/assets/20163971/26dfe1f0-b25b-45df-b3c7-104d2f39d9d9)
Just execute `getFilteredItems` inside this component

Reply:
This method have been moved inside the compontent.